src/main/scala/org/broadinstitute/hail/variant/VariantSampleMatrix.scala

```diff
@@ -229,7 +229,7 @@ object VariantSampleMatrix {
     }
 
     )
-    new VariantSampleMatrix[Genotype](metadata, rdd.toOrderedRDD[Locus])
+    new VariantSampleMatrix[Genotype](metadata, rdd.asOrderedRDD[Locus])
```

Comment:
Yes.  We aren't automating the Kudu tests so this doesn't get tested.  I think this can be optimized by taking advantage of Kudu partitioning.  @tomwhite